data/sheragi outfits.txt

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Outfits
+
+outfit "Electromagnetic Battery"
+	category "Power"
+	cost 1000000
+	thumbnail "outfit/embattery"
+	"mass" 150
+	"outfit space" -150
+	"shield generation" 0.8
+	"shield energy" 0.8
+	"shield heat" 0.8
+	"energy capacity" 20000
+	description `A massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield system of the ship. Sheragi shield systems seems very underdeveloped otherwise.`
+
+outfit "Fusion Drive"
+	category "Engines"
+	cost 80000000
+	thumbnail "outfit/fusiondrive"
+	"mass" 480
+	"outfit space" -480
+	"engine capacity" -230
+	"special slot" 2
+	"jump speed" 0.2
+	"hyperdrive" 1
+	"energy generation" 54
+	"heat generation" 156
+	"turn" 1920.0
+	"turning energy" 3.5
+	"turning heat" 9.5
+	"thrust" 190
+	"thrusting energy" 11
+	"thrusting heat" 33
+	"reverse thrust" 32
+	"reverse thrusting energy" 3.5
+	"reverse thrusting heat" 9.5
+	"flare sprite" "effect/fdrive"
+		"frame rate" 30
+		"rewind"
+	"flare sound" "atomic huge"
+	unplunderable 1
+	description `The Sheragi engines are much more than simple tubes spitting fire; They are a collection of interdependent devices, with increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor and the most powerful plasma engines you've ever seen. Simple technology, but enhanced to the extreme. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further, and allowing the Emerald Sword to fly at impressive speeds. This piece of engineering is what allowed it to travel interstellar space with relative ease.` 
+
+outfit "Sheragi Cooling Unit"
+	category "Systems"
+	cost 1000000
+	thumbnail "outfit/sheragicooling"
+	"mass" 40
+	"outfit space" -40
+	"active cooling" 142
+	"cooling energy" 16.2
+	description `Sheragi cooling systems are primitive at best, but it's active, allowing it to dissipate even more heat than a simple radiator would. It drains a lot of power when the subject to the heat of battle, though.`
+
+outfit "Broken Ship Systems"
```

Comment:
Creative way to prevent the usage of the jump drive. Maybe change to a less generic name like `Damaged Sheragi Systems`?